docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -146,13 +146,13 @@ To disable caching for all Kotlin tasks, set the system property `kotlin.caching
 
 > The Gradle configuration cache is supported only by the following Gradle plugins:
 >   * `org.jetbrains.kotlin.jvm`
->   * `org.jetbrains.kotlin.js`
+>   * `org.jetbrains.kotlin.multiplatform`
```

Comment:
I think at this point it's fair to say that all plugins support GCC, so there's no need to mention it at all. I suggest to remove this note. Of course, not all tasks/actions will be fully GCC-optimized, but that's a usual situation